blog-service/2024-01-15-ui.md

```diff
@@ -0,0 +1,18 @@
+---
+title: New and Improved Sumo Logic UI
+image: https://www.sumologic.com/img/logo.svg
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Update - We've rolled out our new and improved Sumo Logic user interface (UI) with a simplified navigation. Here are some ways we’ve streamlined the overall experience:
+
+* Access all Observability and Security features (like SIEM and SOAR) in one unified navigation. essential to operate and secure modern digital apps.
+* To streamline navigation of the above features, we've moved Administration, Configuration, Help, and User Preferences, to the global application toolbar located on the top right side navigation.
+* Eliminated in-app tabs for a more straightforward and responsive navigation, faster performance. This decision was influenced by years of valuable feedback from Sumo Logic users who highlighted usability issues with the in-app tabs and preferred a consistent browsing experience similar to other web applications. Now, you can utilize browser tabs, session history, the back button, favorites, and other native browser capabilities you're familiar with.
+* URLs are now Stateful, making it easier for you to share content with your teammates. Any changes you make in your UI will be reflected in the URL parameters. This allows you to easily share the exact state of the page with your teammates.
+* Your Recents and Favorite items are now pre-filtered based on the area of the product you are accessing.
+
+For further assistance, contact [Support](https://support.sumologic.com/support/s/).<!-- We'd love your feedback - don't hesitate to get in touch with us at ux-feedback@sumologic.com.-->
```

Comment:
@agaur Should we include the UX Feedback address?

Reply:
@kimsauce I just updated the methods to provide feedback in the doc: https://docs.google.com/document/d/1c71TUwy39J9tHyZnCTsapv4l1XolUAf-1gnS4IpqUDs/edit#heading=h.6egddfrmjovz 
Please use that.